### PR TITLE
Added support for resource-groups

### PR DESCRIPTION
**Pull request by usmangani1**
Added support for resource -groups
https://github.com/localstack/localstack/issues/3979

### CODE REVIEW

File: localstack/services/resourcegroups/rg_starter.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from localstack.services.infra import start_moto_server
+from localstack import config
+
+
+def start_rg(port=None, asynchronous=False, update_listener=None):
+    port = port or config.PORT_RESOURCE_GROUPS
+
+    return start_moto_server('resource-groups', port, name='Resource Groups Tagging API',
```
Comment (whummer) [inline localstack/services/resourcegroups/rg_starter.py:8]: nit: `Resource Groups Tagging API` -> `Resource Groups API`

File: tests/integration/test_resourcegroups.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
+import unittest
+import json
+from localstack.utils.aws import aws_stack
+from localstack.utils.common import short_uid
+
+
+class TestRESROUCEGROUPIntegrations(unittest.TestCase):
```
Comment (whummer) [inline tests/integration/test_resourcegroups.py:7]: nit: could be renamed to `TestResourceGroups`

File: tests/integration/test_resourcegroups.py [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+    def setUp(self):
+        self.resource_group_client = aws_stack.connect_to_service('resource-groups')
+
+    def cleanUp(self, name):
+        self.resource_group_client.delete_group(GroupName=name)
+
+    def test_create_group(self):
+        name = 'resource_group-{}'.format(short_uid())
+        response = self.resource_group_client.create_group(
```
Comment (whummer) [inline tests/integration/test_resourcegroups.py:16]: nit: Looks like lines 16-31 are duplicated 4 times in this class - can we either put this common logic into a util method, or simply combine the 4 test methods into a single test method? (I think testing create/get/list/delete in a single test method would make sense in this case..)

**Review by whummer — COMMENTED** [review]
LGTM, thanks @usmangani1 . 👍  Added a few minor comments. Thanks!

File: tests/integration/test_resourcegroups.py (hunk 2 above)
Comment (usmangani1) [reply tests/integration/test_resourcegroups.py]: Done

File: tests/integration/test_resourcegroups.py (hunk 3 above)
Comment (usmangani1) [reply tests/integration/test_resourcegroups.py:13]: Done

File: localstack/services/resourcegroups/rg_starter.py (hunk 1 above)
Comment (usmangani1) [reply localstack/services/resourcegroups/rg_starter.py]: Done

**Pull request merged**